idea-plugin/src/main/kotlin/arrow/meta/plugin/idea/plugins/comprehensions/comprehensionsIdePlugin.kt

```diff
@@ -15,7 +15,7 @@ val IdeMetaPlugin.comprehensionsIdePlugin: Plugin
     meta(
       addLineMarkerProvider(
         icon = ArrowIcons.BIND,
-        matchOn = { it.safeAs<KtExpression>()?.takeIf(KtExpression::isBinding) },
+        transform = { it.safeAs<KtExpression>()?.takeIf(KtExpression::isBinding) },
```

Comment:
With the current implementation this adds a line marker for the first child of a class.
- this child is not necessarily a leaf
- this child is not necessarily on the same line as the class name identifier (e.g. with annotations)